modules/subscription/variables.tf

```diff
@@ -155,9 +155,9 @@ subscription_tags = {
 DESCRIPTION
   default     = {}
   validation {
-    error_message = "Tag values must contain neither `<>%&\\?/` nor control characters, and must be between 0-256 characters."
+    error_message = "Tag values must be between 0-256 characters."
     condition = alltrue(
-      [for _, v in var.subscription_tags : can(regex("^[^<>%&\\?/[:cntrl:]]{0,256}$", v))]
+      [for _, v in var.subscription_tags : can(regex("^.{0,256}$", v))]
```

Comment:
The old regex excluded the special characters per the docs. Do we still care about that? https://techcommunity.microsoft.com/t5/itops-talk-blog/tags-an-easy-way-to-gain-insight-into-your-cloud-resources/ba-p/846412#:~:text=1%20Azure%20currently%20supports%20up%20to%2050%20tags,a%20maximum%20of%2050%20tag%20name%2Fvalue%20pairs.%20

Reply:
Only Tag names don't support special chars, as it says in the doc also. Tag values only have the length limit

Reply:
![image](https://github.com/Azure/terraform-azurerm-lz-vending/assets/77284962/12cab713-7dd6-46c9-8e07-35ebd97d5d25)

Reply:
Ah, I see. So the value is fine.